### PR TITLE
Feature/redesign detail page

### LINKED ISSUE #185: Redesign van detail page omzetten in code
Branch name: [Copy the exact name of the branch]
Part of: [Story reference]
Blocked by: [Task reference]

**Omschrijving:**

### PR DESCRIPTION
**Pull request by sreen020**
# Description

Closes: #185 

Branch name: feature/redesign-detail-page

_[Please include a summary of the change and which issue is fixed]_

## Type of change

Please select the option that is relevant

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

# Checklist

- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] All the required documentation is added.
- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.

# Additional information

_[Provided extra information if necessary]_

### CODE REVIEW

File: src/lib/GamePage/Accordion.svelte [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+		</ul>
+	{/if}
+{/each}
+
+<style>
+	button {
+		background: transparent;
+		border: 2px solid var(--color-light-orange);
+		width: 100%;
+		padding: 0.7rem 1rem;
```
Comment (jochemvogel) [inline src/lib/GamePage/Accordion.svelte:24]:
```suggestion
		padding: 0.7em 1em;
```
Automatisme zeker, maar hadden volgens mij afgesproken om geen `rem` meer te doen in dit project.

**Comment by jochemvogel** [comment]
En ik denk even nogmaals pushen ofzo om die CI error te fixen. Is iets met Netlify

File: src/lib/GamePage/Accordion.svelte (hunk 1 above)
Comment (iamDistractech) [reply src/lib/GamePage/Accordion.svelte:24]: ik gebruik zowel em als rem, want ze hebben beide een use case

Comment (iamDistractech) [reply src/lib/GamePage/Accordion.svelte:24]: dat is wat we afgesproken hadden (dacht ik?)

Comment (sreen020) [reply src/lib/GamePage/Accordion.svelte:24]: Ja volgens mij ook

**Review by iamDistractech — APPROVED** [review]

**Pull request merged**